Formula/mp.rb

```diff
@@ -0,0 +1,13 @@
+class Mp < Formula
+  desc "Minimum Profit editor by Angel Ortega"
```

Comment:
Don't use formula name in the description, and could you please make it more descriptive

Reply:
Edited it to make it more descriptive.

Reply:
...and removed the formula name.